querybook/webapp/lib/sql-helper/sql-autocompleter.ts

```diff
@@ -287,10 +313,17 @@ export class SqlAutoCompleter {
                 const tableNames = await this.getTableNamesFromPrefix(prefix);
 
                 for (const tableName of tableNames) {
-                    const match = tableName.match(/^(\w+)\.(\w+)$/);
-                    if (match) {
+                    const schemaTableNames = tableName.split('.');
+
+                    if (schemaTableNames.length === 2) {
```

Comment:
I feel like I saw this similar code piece multiple times, might be good to have tiny util function for it? can return an object with parsed schema name and table name, so that

```
const {schemaName, tableName} = parseFullTableName(..)
if (schemaName && tableName) {
   ...
   this.flatQuotationFormatter(tableName)
   ...
}
```

Reply:
hmm it is just `.split('.')` and used in 2 places, i think we can make a function for it if it is more common in future